app/assets/javascripts/angular/components/organizationCleanup/organizationCleanup.component.js

```diff
@@ -22,9 +22,38 @@ function organizationCleanupController(
     this.archiveInactivityDate = new Date();
     this.checkIcon = checkIcon;
 
+    const showConfirmModal = async title => {
+        const archiveValue =
+            title === 'ministries.cleanup.archive_by_inactivity_description'
+                ? 'leaders_last_sign_in_at'
+                : 'persons_inactive_since';
+        const archiveDate =
+            title === 'ministries.cleanup.archive_by_inactivity_description'
+                ? this.archiveInactivityDate
+                : this.archiveBeforeDate;
+
+        const confirmModal = $uibModal.open({
+            component: 'iconModal',
+            resolve: {
+                title: () => $filter('t')(`${title}`),
```

Comment:
Title should already be a string...
```suggestion
                title: () => $filter('t')(title),
```